packages/expo-dev-client-components/README.md

```diff
@@ -0,0 +1,13 @@
+# expo-dev-client-components
+
+Describe your module and why it's important in a few sentences.
+
+## API documentation
+
+## Installation in managed Expo projects
+
+## Installation in bare React Native projects
+
+## Contributing
+
+Contributions are very welcome! Please refer to guidelines described in the [contributing guide](https://github.com/expo/expo#contributing).
```

Comment:
Please, fill in the readme file (if you're doing it in separate PR, ignore this comment)😅